src/components/todoerr/todoerr.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+import { useTodos } from '../../context/todoProvider';
+
+export const TodoErr = () => {
+  const { error, setError } = useTodos();
+
+  useEffect(() => {
+    setTimeout(() => {
+      setError(null);
+    }, 3000);
+  }, [error, setError]);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal', {
+          hidden: !error,
+        },
+      )}
```

Comment:
// {/* Add the 'hidden' class to hide the message smoothly */}
  it's mean, than you need to add transition for styled as smoothly